Copy updates

jasonLaster opened the pull request:
This update incorporates many of the ideas in Brian's doc and landing page.

---
**Goals**
1. Make the hero more descriptive so that it's clear that Replay is a browser that helps devs retroactively debug their app
2. Update section headers to be more user centric so that it's more likely users appreciate how Replay can help them. 
3. Make section sub headers more descriptive so it's more likely they'll understand how Replay works
4. Make the section content results oriented so that Replay's impact is more clear

---

**Observations**

1. **Each section's job is to get the user to the next section.** I tried to have each section do the one thing that would help the user get to the next section. 

> Roughly that is: Replay is a browser that's able to let you debug your application after the fact. You're going to love how Replay lets you record a bug once and never reproduce it again. This helps in two big ways. First it's the ultimate bug reporting tool. Second it lets you finally get your test suite under control. Now that you get it, let me tell you a bit more about how amazing the debugging environment is. It's pretty fucking cool. Oh and by the way, unlike every other debugging tool before it, Replay was designed with collaboration at the start. And in case you're worried about whether Replay works at your company, we're enterprise secure and ready.

2. **It's important that the story flows** I want to push back against listing the 4 pain points directly because I think it's more important that we bring the reader along the story as they understand how replay works and how it will help them. The biggest part of this story is talking about bug reports and test suites as part of the same shared reproducibility problem.


---

**Next steps**
Of course these changes are not all that needs to be done. The goal here is to simply make the story more user centric. After we hit that we will

1. Improve the CTAs so that the funnel is stronger
2. Improve the Workflows dropdown and bug reports and test suites sub pages 

---

**Asides**

Just want to acknowledge how difficult it is to tell the story. I've spent probably 10 hours on this version and have tried hundreds of different variants. Really appreciate all the help, suggestions, etc. Excited to see what this new version will look like

bhackett1024 commented:
This PR is pretty disappointing to me.  The hero message is still focused on the tool's features, there is nothing here to speak to the economic buyer, this slightly reworks the existing website without addressing its critical shortcomings.

jasonLaster commented:
Updated a couple of additional items

1. updated the hero to be user centric. Note there's now a variant system so we could A/B test 
2. Updated the reproducibility H2 to be more directly about reproducibility

I did not make larger layout changes since I'd like to first land the content changes which are a step in the right direction.

bhackett1024 commented:
The new hero is a lot better.  I feel that "fix the hardest issues" is still only getting at part of the value prop though.  Replay has such a range of uses that we'll need to continue iterating on the hero to strike the right balance to avoid being too vague, too long winded, and pigeonholing ourselves.  It's good we can A/B test different heros.
